src/components/application_manager/include/application_manager/policies/policy_handler.h

```diff
@@ -422,6 +420,18 @@ class PolicyHandler : public PolicyHandlerInterface,
    */
   uint32_t GetAppIdForSending() const OVERRIDE;
 
+  /**
+   * @brief Add application to PTU queue if don't have application
+   * with the same app_id
+   * @param new_app_id app id new application
+   */
+  void AddNewApplicationIdToPTUQueue(const uint32_t new_app_id);
```

Comment:
@ydementieiev can be renamed to `PushAppIdToQueue` and `PopAppIdFromQueue`

Reply:
@AKalinich-Luxoft done in [fe7b393](https://github.com/smartdevicelink/sdl_core/pull/3208/commits/fe7b393fe80f89ac9137d6dd9b41a0dfd6ba28c6)